bus-mapping/src/rpc.rs

```diff
@@ -239,7 +294,8 @@ mod rpc_tests {
         const TARGET_PROOF: &str = r#"{
             "address": "0x7f0d15c7faae65896648c8273b6d7e43f58fa842",
             "accountProof": [
-                "0xf873a12050fb4d3174ec89ef969c09fd4391602169760fb005ad516f5d172cbffb80e955b84ff84d8089056bc75e2d63100000a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a0c5d2460186f7233c927e7db2dcc703c0e500b653ca82273b7bfad8045d85a470"
+                "0xf9011180a0ab8cdb808c8303bb61fb48e276217be9770fa83ecf3f90f2234d558885f5abf18080a01a697e814758281972fcd13bc9707dbcd2f195986b05463d7b78426508445a04a01f5061d0eb56546aa17ec23486841e6cf073d98069c4b41c3fc9a0dc1cab0c89a0929ef457bb59f8a38d431af8d917ce82a8b608bd50edfc2b4a04f8b7bd7c583580a02e0d86c3befd177f574a20ac63804532889077e955320c9361cd10b7cc6f580980a064b03ddf9f38fbbf8d0cf12171626ed809fcc99d85e30fe6a2cca7a0ee7475178080a01b7779e149cadf24d4ffb77ca7e11314b8db7097e4d70b2a173493153ca2e5a0a066a7662811491b3d352e969506b420d269e8b51a224f574b3b38b3463f43f0098080",
+                "0xf851a070ebfba8dc7e9dd75891224e8cce5b757c74f9879d7d4a2665e7e422aa0aed4480808080808080808080a0e61e567237b49c44d8f906ceea49027260b4010c10a547b38d8b131b9d3b6f848080808080"
```

Comment:
I'm not sure that this will not change between Geth docker instances.

@ed255 WDYT?

Reply:
Since we always use the same account, this shouldn't change from one run to the other.  Nevertheless, since the queried block is the latest, every time we add more transactions this will change, so I think it would be better to use a specific block number.